ili/dataloaders/loaders.py

```diff
@@ -80,11 +85,11 @@ def get_all_parameters(self):
 
 
 class StaticNumpyLoader(NumpyLoader):
-    """Class to load single numpy files of summaries and parameters
+    """Loads single numpy files of data and parameters from file
```

Comment:
Repeated "files" in sentence - could be worded better

Reply:
Fixed.